### PR TITLE
PR: Add templates to pip package

### LINKED ISSUE #67: Templates not included in pip package
After a `pip install`, there are no templates in `.../loghub/templates':
```
$ ls site-packages/loghub/templates/
__init__.py  __pycache__/
```
This leads to a `FileNotFoundError` in `render_changelog`

### PR DESCRIPTION
**Pull request by goanpeca**
Fixes #67

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -53,6 +53,7 @@ def get_description():
     description='Generate changelogs based on Github milestones or tags',
     long_description=get_description(),
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
+    package_data={'loghub.templates.content': ['*.txt']},
```
Comment (jitseniesen) [inline setup.py]: Why `content` ?

Comment (goanpeca) [reply setup.py]: oops!

**Review by jitseniesen — CHANGES_REQUESTED** [review]
Need to fix directory name

**Comment by goanpeca** [comment]
@jitseniesen done, I copied the code from another repo and that was a leftover :-). I am finishing a PR to include an improvement on loghub so I can make release 0.3 (which will fix this)

**Review by jitseniesen — APPROVED** [review]

**Comment by goanpeca** [comment]
@ccordoba12 should I issues 0.2.1 that just fixes this, or go ahead and release 0.3 with the new extra links PR?

**Comment by ccordoba12** [comment]
Please release 0.2.1 to have a 0.2 working release (because 0.2.0 was broken, if I understand correctly).

Then release 0.3.

**Pull request merged**